### DISCUSSION TITLE
Change my Figma name

### DISCUSSION THREAD

**Discussion by pamelahu** [twilio-labs/paste#2055] [General] [answered]
Can my name get changed to "Pamela Hu" in Figma please?
<img width="373" alt="Screen Shot 2021-11-17 at 1 33 36 PM" src="https://user-images.githubusercontent.com/18605252/142261503-921b0534-8626-47bf-b714-579e5abeeecb.png">

(Should I be posting this type of question into the discussion?)

**Comment by serifluous** [comment, marked as answer]
Hey @pamelahu! You should be able to change it on your own through your settings: https://help.figma.com/hc/en-us/articles/360041034433-Manage-name-or-avatar#Update_account_name

If you aren't able to do it, it's something that's likely managed by IT rather than us within Figma.
